### ISSUE TITLE
$ yo meanjs:angular-module <module-name> - Docs on the site are out of date

### ISSUE THREAD

**Issue by rohanleach4** [meanjs/generator-meanjs#208]
$ yo meanjs:angular-module <module-name>

no longer is correct. I believe it should be:

$ yo meanjs:mean-module <module-name>

**Comment by simison** [comment]
Wanna send a PR? :-)

**Comment by rohanleach4** [comment]
I would love to, but in this case, I am not sure how to :-/

**Comment by nodox** [comment]
I just ran into this problem. Would love to have updated docs. @rohanleach4 do you need help creating a pull request?

**Comment by rohanleach4** [comment]
@nodox yes please. That would be a great help

**Comment by nodox** [comment]
@rohanleach4  [Here is a great tutorial on how to create PR](https://yangsu.github.io/pull-request-tutorial/)

tl;dr
1. Create local copy,
2. pull from origin,
3. create a local branch for the fix
4. code, commit, push branch with fix to origin
5. create PR on PR page with your branch.
 
The first half is all you really need but the bits towards the end are useful once you start contributing more.

**Comment by rohanleach4** [comment]
Sorry for my tardiness of reply - stacked out at the moment.
I will look into this over this coming week - thanks for your time :-)
